src/main/java/uk/gov/hmcts/reform/wataskmonitor/domain/taskmonitor/request/JobDetails.java

```diff
@@ -11,7 +12,7 @@
     private final JobName name;
 
     @JsonCreator
-    public JobDetails(JobName name) {
+    public JobDetails(@JsonProperty("name") JobName name) {
```

Comment:
This should not be needed since we use the ParameterNamesModule in Jacksonconfig

Reply:
as I remember I had to add this because there was a IT failure